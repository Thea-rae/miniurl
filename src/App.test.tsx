import React from 'react'
import App from './App'
import { create } from 'react-test-renderer'
import { render , screen } from '@testing-library/react'

describe('snapshot test', () => {
  test('looks at the whole app', () => {
    let tree = create(<App />)
    expect(tree.toJSON()).toMatchSnapshot()
  })
})

describe('App', () => {
  test('renders App component', () => {
    render(<App />)
    // screen.debug()
  })
})

