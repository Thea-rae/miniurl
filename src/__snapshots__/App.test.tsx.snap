// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test looks at the whole app 1`] = `
<div
  className="MuiContainer-root MuiContainer-maxWidthLg"
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
  >
    <h2
      className="MuiTypography-root MuiTypography-h2"
    >
      MiniUrlz
    </h2>
  </div>
  <div
    className="MuiGrid-root"
  >
    <div
      className="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
      style={
        Object {
          "marginTop": "2em",
        }
      }
    >
      <div
        className="MuiCardContent-root"
      >
        <form
          action="#"
          onReset={[Function]}
          onSubmit={[Function]}
        >
          <div
            className="MuiFormControl-root MuiTextField-root"
            style={
              Object {
                "width": "100%",
              }
            }
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink={false}
              htmlFor="url-input"
              id="url-input-label"
            >
              Url to shrink
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              onClick={[Function]}
            >
              <input
                aria-describedby="url-input-helper-text"
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiInput-input"
                disabled={false}
                id="url-input"
                name="url"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={false}
                type="url"
                value=""
              />
            </div>
            <p
              className="MuiFormHelperText-root"
              id="url-input-helper-text"
            >
              *required
            </p>
          </div>
          <div
            className="MuiFormControl-root MuiTextField-root"
            style={
              Object {
                "width": "100%",
              }
            }
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink={false}
              htmlFor="slug-input"
              id="slug-input-label"
            >
              Custom Slug
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              onClick={[Function]}
            >
              <input
                aria-describedby="slug-input-helper-text"
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiInput-input"
                disabled={false}
                id="slug-input"
                name="slug"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={false}
                type="text"
                value=""
              />
            </div>
            <p
              className="MuiFormHelperText-root"
              id="slug-input-helper-text"
            >
              *optional
            </p>
          </div>
          <button
            className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
            disabled={false}
            id="submit-button"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            style={
              Object {
                "marginTop": "3em",
                "width": "100%",
              }
            }
            tabIndex={0}
            type="submit"
          >
            <span
              className="MuiButton-label"
            >
               Shrink 
            </span>
          </button>
        </form>
      </div>
    </div>
  </div>
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary mui-fixed MuiPaper-elevation4"
    style={
      Object {
        "bottom": 0,
        "top": "auto",
      }
    }
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
    >
      <span
        className="MuiTypography-root MuiTypography-caption"
      >
        Made for the homies in 2020
      </span>
    </div>
  </header>
</div>
`;
